assets/agenda/actions.js

```diff
@@ -675,3 +686,16 @@ export function stopWatchingCoverage(coverage, item) {
             }, (error) => { errorHandler(error, dispatch);});
     };
 }
+
+export function errorHandler(error, dispatch, setError) {
+    console.error('error', error);
+    if (setError) {
+        if (error.response && error.response.status === 403) {
+            dispatch(setErrorMessage('No Products'));
```

Comment:
imo it would make sense to set here the actual message using gettext and then only display `status.errorMessage` in the UI so we can handle different response codes if needed, storing there `No Products` which is not checked later on makes it rather confusing.